orchagent/portsorch.cpp

```diff
@@ -3851,6 +3851,17 @@ void PortsOrch::doLagMemberTask(Consumer &consumer)
                     continue;
                 }
 
+                if (!port.m_ingress_acl_tables_uset.empty() || !port.m_egress_acl_tables_uset.empty())
```

Comment:
My concern is, this only prevents from ASIC but kernel/teamd will have this port part of Portchannel, right?

Reply:
@prsunny correct, but imho this is a common practise in SONiC

Reply:
Not exactly, this leaves the device in inconsistent state.  Whereas either SAI accepting the config or erroring out and exiting orchagent could help in catching the issue sooner.

Reply:
@prsunny without this check the device will be also in inconsistent state in terms of ACL/PBH. What can you suggest here?